src/utils/crypto.ts

```diff
@@ -6,10 +6,14 @@
  */
 
 import CryptoJS from 'crypto-js';
+import bitcore from 'bitcore-lib';
 import { DecryptionError, InvalidPasswdError, UnsupportedHasherError } from '../errors';
-import { HASH_ITERATIONS, HASH_KEY_SIZE } from '../constants';
+import { HATHOR_MAGIC_BYTES, HASH_ITERATIONS, HASH_KEY_SIZE } from '../constants';
 import { IEncryptedData } from '../types';
 
+// Monkey-patch MAGIC_BYTES to use Hathor's
+bitcore.Message.MAGIC_BYTES = Buffer.from(HATHOR_MAGIC_BYTES);
```

Comment:
> We can and probably should use the new utils/crypto.ts method, but I suggest we don't refactor it in this PR as we also need to change wallet-service to expect Hathor Signed Message\n instead of Bitcoin Signed Message\n it currently expect.

That's my main concern. If we changed the magic bytes for the lib, the current wallets we have in the wallet service wouldn't work anymore, right? Because they were generated using the old magic bytes, or not?

Reply:
I checked and this is indeed a problem, since we are monkey patching the Message class, it will affect the signMessage for the wallet-service.

I will merge this and work on a fix on the wallet-service side

Reply:
@pedroferreira1 PR opened in https://github.com/HathorNetwork/hathor-wallet-service/pull/426